Packs/Phishing/Playbooks/Phishing_-_Generic_v3.yml

```diff
@@ -525,6 +537,9 @@ tasks:
           accessor: reportedemailfrom
           transformers:
           - operator: ExtractEmailTransformer
+      O365AllowNotFoundExchangeLocations:
+        complex:
+          root: inputs.O365AllowNotFoundSearchLocations
```

Comment:
Is it possible that here you are partially fixing https://github.com/demisto/content/pull/21190 too?